### PR TITLE
Fix complex import manual

### PR DESCRIPTION
**Pull request by violinyanev**
This is just updating to the current behavior. We should improve the docs further, but will do that as a separate PR.

### CODE REVIEW

File: complex_import/manual.md [hunk 1]
```diff
@@ -58,32 +58,50 @@ To make it easier to match objects after the import, the naming convention for a
 
 * Cylinder/TX+2   (a cylinder object which is translated along the X axis by 2 units in the positive direction)
 * Cone/TY+5/RX-90  (a cone which is translated along Y by +5 units, and rotated around X by -90 degrees)
+* Meshes have a 'Mesh' suffix in the name, Materials have a 'Mat_' prefix
```
Comment (bojackHaasman) [inline complex_import/manual.md:61]: Is it also 'Mesh_' with underscore like 'Mat_' or 'Mesh'?

File: complex_import/manual.md [hunk 2]
```diff
@@ -58,32 +58,50 @@ To make it easier to match objects after the import, the naming convention for a
...
 multiple nodes arranged hierarchically, it's not
 convenient to import these one by one. The Ramses Composer supports the import of an entire scene graph
 over the `Import GLTF Assets` function, which is available
-in the `File` menu or via right click on the `Scene Graph` or `Resources` views. The option prompts the
-selection of a glTF file to import, then proceeds to perform
-a one-time import of all nodes and meshes in the specified file. After the import is finished, the result looks something like this:
+via right click on any node in the `Scene Graph` view. See the
+[details section on glTF import](#gltf-import-details) for more details.
+
+In this example, we will import all content from the `structure.gltf` file which was
```
Comment (bojackHaasman) [inline complex_import/manual.md:80]:
```suggestion
In this example we will import all content from the `structure.gltf` file which was
```

File: complex_import/manual.md [hunk 3]
```diff
@@ -58,32 +58,50 @@ To make it easier to match objects after the import, the naming convention for a
...
 
 ![](./docs/camera_params.png)
 
+Adapt the root node too:
+
+![](./docs/node_params.png)
+
+Since this example uses transparent objects which are order-dependent, you need a deterministic
+ordering mechanics to ensure consistent rendering. Select the MainRenderLayer object in the Resources
+menu and choose the option 'Scene graph order' in the `Render Order` field:
```
Comment (bojackHaasman) [inline complex_import/manual.md:99]:
```suggestion
menu and choose the option `Scene graph order` in the `Render Order` field:
```

File: complex_import/manual.md [hunk 4]
```diff
@@ -163,14 +184,10 @@ As soon as the glTF file changed, you can immediately observe the new geometry i
 ![](./docs/voxel_modifier_5.png)
 
 You can arbitrarily modify all meshes' geometry and re-export - and the mesh's changes will be automatically imported by the Composer.
-There is a limitation though: the nodes and their transformations will not be re-imported. If you move an object or rotate it differently - this change
-will not be re-imported by the Composer. As a workaround, you can import the entire file again, copy-paste only the nodes that you want re-imported, then delete
+Note that nodes and their transformations will not be re-imported. If you move an object or rotate it differently - this change
+will not be re-imported by the Composer. You can import the entire file again, copy-paste only the nodes that you want re-imported, then delete
```
Comment (bojackHaasman) [inline complex_import/manual.md:188]:
```suggestion
will not be re-imported by the Composer. You can import the entire file again, copy-paste only the nodes that you want re-imported and then finally delete
```

File: complex_import/manual.md [hunk 5]
```diff
@@ -184,9 +201,37 @@ a material assigned to them and you have to export materials in the glTF export
 Also worth noting - after import, the material settings (shader uniforms) are taken over from the material instance in the Composer. If you want to take over
 existing settings, you have to manually transfer them (or copy-paste the newly imported MeshNode over the existing one).
 
-## A note on negative scale values
+## glTF import details
+
+The glTF import option prompts the
+selection of a glTF file to import, then proceeds to perform
+a one-time import of all nodes and meshes from the specified file. This option requires
```
Comment (bojackHaasman) [inline complex_import/manual.md:208]:
```suggestion
a one-time import of all nodes and meshes from that specified file. This option requires
```

**Review by bojackHaasman — CHANGES_REQUESTED** [review]

File: complex_import/manual.md [hunk 6]
```diff
@@ -58,32 +58,50 @@ To make it easier to match objects after the import, the naming convention for a
...
 You must adapt the default camera parameters to have a better view of the scene:
 
 ![](./docs/camera_params.png)
 
+Adapt the root node too:
+
+![](./docs/node_params.png)
+
+Since this example uses transparent objects which are order-dependent, you need a deterministic
+ordering mechanics to ensure consistent rendering. Select the MainRenderLayer object in the Resources
```
Comment (ParadoxDaKo) [inline complex_import/manual.md:98]:
```suggestion
order to ensure consistent rendering. Select the MainRenderLayer object in the Resources
```

"a deterministic order" - should also make sense?

File: complex_import/manual.md [hunk 7]
```diff
@@ -163,14 +184,10 @@ As soon as the glTF file changed, you can immediately observe the new geometry i
 ![](./docs/voxel_modifier_5.png)
 
 You can arbitrarily modify all meshes' geometry and re-export - and the mesh's changes will be automatically imported by the Composer.
-There is a limitation though: the nodes and their transformations will not be re-imported. If you move an object or rotate it differently - this change
-will not be re-imported by the Composer. As a workaround, you can import the entire file again, copy-paste only the nodes that you want re-imported, then delete
+Note that nodes and their transformations will not be re-imported. If you move an object or rotate it differently - this change
+will not be re-imported by the Composer. You can import the entire file again, copy-paste only the nodes that you want re-imported, then delete
 the second copy of the import.
```
Comment (ParadoxDaKo) [inline complex_import/manual.md:190]: With the new asset import dialog, the user does not need to reimport "the entire file" anymore - worth mentioning & changing the paragraph here?

**Review by ParadoxDaKo — CHANGES_REQUESTED** [review]
The general gist of the info has been properly updated - added some comments.

File: complex_import/manual.md (hunk 1 above)
Comment (violinyanev) [reply complex_import/manual.md:61]: Nope, it's just as it says. Not sure if worth the effort to change... It's a lot of work

File: complex_import/manual.md [hunk 8]
```diff
@@ -184,9 +201,37 @@ a material assigned to them and you have to export materials in the glTF export
...
+
+* cameras
+* animations
+* images/samples/textures
+* materials
 
-Currently, Ramses Composer uses [assimp](https://github.com/assimp/assimp) to import glTF files and convert the glTF data to Ramses Composer data. During testing, we found out that when encountering negative scale values during a node import, Assimp flips all other positive scale values of that node and compensates with respective 180 degree rotations. This approach does not harmonize with how Ramses handles node scaling and rotation. As a preventive measure, importing nodes with negative scale values is temporarily prohibited in Ramses Composer. We are currently looking into replacing assimp with a more one-to-one-import-friendly glTF importer.
+Even though materials are not supported, there is a special behavior to use material/shaders created in the
+Ramses Composer project. If a MeshNode N references a material M in the glTF file, and a material with name M
+exists in the Ramses Composer project, then the imported node N will automatically reference material M.
```
[the remark this answers is not in the document]
Comment (violinyanev) [reply complex_import/manual.md:232]: Your suggestion isunfortunately wrong - it is _mesh_ which holds a reference to materials, indirectly over its primitives attribute:
https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#reference-mesh
https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#reference-mesh-primitive

[the remark this answers is not in the document]
Comment (ParadoxDaKo) [reply complex_import/manual.md:232]: True, so my suggestion is now:

```suggestion
Ramses Composer project. If a mesh primitive M references a material N in the glTF file, and the Ramses Composer scene
contains a Material with name N, then every imported MeshNode that references the mesh primitive M will automatically reference the material N.
```

**Pull request merged**